pkg/archive/archive_windows.go

```diff
@@ -28,8 +28,7 @@ func CanonicalTarNameForPath(p string) (string, error) {
 // chmodTarEntry is used to adjust the file permissions used in tar header based
 // on the platform the archival is done.
 func chmodTarEntry(perm os.FileMode) os.FileMode {
-	// Clear r/w on grp/others: no precise equivalen of group/others on NTFS.
-	perm &= 0711
+	perm &= 0755
 	// Add the x bit: make everything +x from windows
 	perm |= 0100
```

Comment:
you need to make this `0111` since we'll add +x to others/grp as well now.

also fix the `TestChmodTarEntry` test in `pkg\archive\archive_windows_test.go` (you don't notice it's failing because it's currently not part of the CI suite).